packages/manager/apps/web/client/app/hosting/email/detach/detach.html

```diff
@@ -0,0 +1,19 @@
+<div data-ovh-alert="detach"></div>
+
+<oui-back-button aria-label="Aria label for heading" on-click="$ctrl.goBack()">
+    <span data-translate="hosting_email_detach_option_title"></span>
+</oui-back-button>
+
+<p
+    data-translate="hosting_email_detach_option_description"
+    data-translate-values="{ 'hosting': $ctrl.serviceName }"
+></p>
+
+<ovh-manager-product-offers
+    pricing-type="$ctrl.pricingType"
+    user="$ctrl.user"
+    on-error="$ctrl.onError(error)"
+    on-success="$ctrl.onSuccess(detachResult)"
+    workflow-type="$ctrl.workflow.type"
+    workflow-options="$ctrl.workflow.options"
```

Comment:
Missing `data-` in all the above attributes :)

Reply:
As @FredericEspiau mentioned in other PR, we could remove the `data-`, for component attributes (and no HTML5 attributes rewroten here)